code/game/objects/items/powerfist.dm

```diff
@@ -35,6 +35,15 @@
 	fire = 100
 	acid = 40
 
+/obj/item/melee/powerfist/proc/fist_pressure_setting_to_text(fist_pressure_setting)
+	switch(fist_pressure_setting)
```

Comment:
Add an `else` case here that calls `CRASH("Invalid pressure setting: [fist_pressure_setting]")